### ISSUE TITLE
make void map near infinite or atleast bigger than humongous

### ISSUE THREAD

**Issue by username90001** [studio-minus/ppg-bugs-and-requests#2732]
### Feature description

void map is toooooooooooooooooooooooo small compared to humongous

### Feature purpose

space
i tried to place solar system that i was building in humongous to void but result it was bigger than map

**Comment by Mesalon** [comment]
I agree. Void should just be... a void. Not a small room with no walls.

**Comment by username90001** [comment]
void is small room we need infinitly big rooms

**Comment by DindinYT37** [comment]
Duplicate of or Related to:
* #676
* #1391
* #2365

**Comment by username90001** [comment]
it is not i mean make map bigger

**Comment by username90001** [comment]
void is way too small
